### ISSUE TITLE
Some questions related to feature selection

### ISSUE THREAD

**Issue by HelloWorldLTY** [gao-lab/GLUE#4]
Hi, sorry to disturb you again. After reviewing your process related to data preprocessing, I am wondering if you utilize the highly variable genes selection process in these steps:
print('Preprocessing GEX...')
gex_prep = utils.GEXPreprocessing(n_comps=100, n_genes=n_genes, merge_adt=omics == "cite")
gex_prep.fit_transform(gex)

if omics == "cite":
    print('Preprocessing ADT...')
    other_prep = utils.ADTPreprocessing(n_comps=100)
elif omics == "multiome":
    print('Preprocessing ATAC...')
    other_prep = utils.ATACPreprocessing(n_comps=100)

Could you please share your ideas about hvg selection with me? If I did hvg selection, will the matching become more accuracy? Thanks a lot.

**Comment by Jeff1995** [comment]
Sorry for the delayed response! Yes we do feature selection in preprocessing. The code is available [here](https://github.com/Jeff1995/neurips2021_multimodal_topmethods/blob/clue/src/match_modality/methods/clue/resources/utils.py). 

Doing feature selection (especially in the GEX domain) does improve accuracy. You may also look at the discussion in #2.

**Comment by HelloWorldLTY** [comment]
Thanks for your help, I think your approach makes sense.

**Comment by Jeff1995** [comment]
Great! You're welcome!
